### ISSUE TITLE
Error Plugin Load Failed

### ISSUE THREAD

**Issue by getrisedotio** [tea3/hexo-generator-amp#17]
I'm starting a hexo blog and followed the installation instructions. 

I copied the following into the __config.yml.

```javascript The following settings is the quick start options.
generator_amp:
  templateDir:  amp-template
  assetDistDir: amp-dist
  logo:
    path:   sample/sample-logo.png
    width:  600
    height: 60
  substituteTitleImage: 
    path:   sample/sample-substituteTitleImage.png
    width:  1024
    height: 800
  warningLog: false   # If you want to validate, please set true.
```

Then if I run ```hexo generate```, I get an error.

> ERROR Plugin load failed: hexo-generator-amp

I'm running Windowx 10 x64 using node v6.9.4.

Any help would be appreciated!

**Comment by tea3** [comment]
@getrisedotio
Thank you for using this plugin .

About this issue , I tried it in this environment. But the same error did not occur. 

- Mac OSX 10.11 (Sorry , I can't tried windows. )
- node v6.9.4
- npm  v4.1.1
- hexo-cli@1.0.2

```
$ nvm install v6.9.4
$ nvm use v6.9.4
$ npm install -g npm
$ npm install -g hexo-cli
$ hexo init hexo694
$ cd hexo694
$ npm install hexo-generator-amp --save
```


I copied the following into the `./_config.yml` .

```
# The following settings is the quick start options. 
generator_amp:
  templateDir:  amp-template
  assetDistDir: amp-dist
  logo:
    path:   sample/sample-logo.png
    width:  600
    height: 60
  substituteTitleImage: 
    path:   sample/sample-substituteTitleImage.png
    width:  1024
    height: 800
  warningLog: false   # If you want to validate, please set true.
```

I started a local server. 

```
$ hexo s
```

Next , I opend [http://localhost:4000/2017/01/24/hello-world/amp/#development=1](http://localhost:4000/2017/01/24/hello-world/amp/#development=1).  But the same error did not occur.


Windows user , do the same problem occur ? Please give me advice.

**Comment by tea3** [comment]
@getrisedotio 

Please refer to the follow. Can you solve it the same way ?
https://github.com/hexojs/hexo/issues/1057

**Comment by getrisedotio** [comment]
Thanks tea3! I did a clean install of of hexo in a new directory and followed your instructions, and it worked! 

```javascript 
$ nvm install v6.9.4
$ nvm use v6.9.4
$ npm install -g npm
$ npm install -g hexo-cli
$ hexo init hexo694
$ cd hexo694
$ npm install hexo-generator-amp --save
```

I went back and saw that my old hexo directory was missing the amp-template directory and the amp-dist directory (in public). I'm not sure how that came to be, but if anyone else has the same issue, take a look to make sure the directories and present!
